pallets/identity/src/lib.rs

```diff
@@ -1159,6 +1162,21 @@ impl<T: Trait> Module<T> {
         None
     }
 
+    pub fn has_valid_kyc(claim_for: IdentityId) -> Option<IdentityId> {
```

Comment:
Should we start using it elsewhere as well? For example when checking for KYC in SignedExtension.

Reply:
I think we should add the genesis config for CDD providers first, otherwise we'll have a bad time testing anything ;-).

Reply:
I agree, we need to add CDD providers first.